core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/PhysicalPlan.scala

```diff
@@ -91,28 +76,26 @@ case class PhysicalPlan(
 
   private def getPhysicalOpForInputPort(
       logicalOpId: OperatorIdentity,
-      portName: String
+      portId: PortIdentity
```

Comment:
remove this function.

Reply:
we will have to keep it because it uses the logical op id.